### PR TITLE
api call and category display

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this PR, 
1. Database is created using mirage js for quizzes.
2. API call is done in questionContext.
3. And data is mapped to show the categories.
Please review and provide feedback

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Overall the code looks good.

File: src/components/categoryCard/categoryCard.jsx [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+    const { quizData } = useQuestion();
+
     return (
-        <div className='card-container'>
-            <img src={historyImg} alt="category" className="category__img" />
-            <h2 className="category--title"> History </h2>
-        </div>
+        quizData.map(quizItem => {
+            return (
+                <div key={quizItem._id}>
```
Comment (SaurabhChawda) [inline src/components/categoryCard/categoryCard.jsx:11]: You can  use Link over here.

**Review by SaurabhChawda — APPROVED** [review]
Overall looks good to me, You can merge it.

**Review by tanishk15sharma — COMMENTED** [review]
I liked how you are using naming conventions , overall code is good to merge

**Pull request merged**